courses/free-courses-en.md

```diff
@@ -373,6 +373,7 @@
 * [Learn HTML5 Programming From Scratch](https://www.udemy.com/learn-html5-programming-from-scratch/)
 * [Learn to style HTML using CSS](https://developer.mozilla.org/en-US/docs/Learn/CSS)
 * [Responsive Web Design with HTML5 and CSS3 - Advanced](https://www.udemy.com/course/responsive-web-design-with-html5-and-css3-advanced/) - Udemy
+* [Responsive Web Design certification] (https://www.freecodecamp.org/learn/responsive-web-design/) - freeCodeCamp
```

Comment:
It seem that there are listed.

https://github.com/EbookFoundation/free-programming-books/blob/54cf035016e84a4d2e126c011498b1cd97c4ddc3/more/free-programming-interactive-tutorials-en.md

Maybe fix wrong link there...

Reply:
Thank you for pointing this out!

I'll be sure to check for duplicates from here on out!!

Reply:
@Gokul-R-Prakash 

![image](https://user-images.githubusercontent.com/3125580/135741424-f0521f97-efec-48a3-8b8a-821cadbcdecf.png)

As you can see all frecodecamp "learn" are in `free-programming-interactive-tutorials-en.md` move it there